core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts

```diff
@@ -86,6 +86,8 @@ export class FloatingActionsView {
         this.command = command;
         this.key = key;
         this.tippyOptions = tippyOptions;
+
+        document.addEventListener('DOMContentLoaded', () => this.setButtonPosition());
```

Comment:
To achieve what Freddy mentioned, we can do this:

```suggestion
        document.addEventListener('DOMContentLoaded', this.setButtonPosition.bind(this));
```